src/snowcli/app/printing.py

```diff
@@ -8,9 +8,9 @@
 from rich import box, get_console, print
```

Comment:
I think printing is a core functionality and should be in `api`. WDYT?

Reply:
Currenlty we don't encourage users to do printing. Once we have established way of showing output other than `with_output` decorator we can put it in `api`. I would treat current `printing` module as internal implementation